packages/keplr/src/wallet-connect/main-wallet.ts

```diff
@@ -0,0 +1,109 @@
+import { ChainName, ChainRegistry, Dispatch, State } from '@cosmos-kit/core';
+import { MainWalletBase } from '@cosmos-kit/core';
+import { KeplrWalletConnectV1 } from '@keplr-wallet/wc-client';
+import WalletConnect from '@walletconnect/client';
+import EventEmitter from 'events';
+
+import { ChainWCKeplr } from './chain-wallet';
+import { WCKeplrData } from './types';
+
+export class WCKeplrWallet extends MainWalletBase<
+  KeplrWalletConnectV1,
+  WCKeplrData,
+  ChainWCKeplr
+> {
+  protected _chains: Map<ChainName, ChainWCKeplr>;
+  protected _client: KeplrWalletConnectV1;
+  connector: WalletConnect;
+  ee: EventEmitter;
+
+  constructor(_concurrency?: number) {
+    super(_concurrency);
+
+    this.connector = new WalletConnect({
+      bridge: 'https://bridge.walletconnect.org'
+    });
+
+    this.connector.on("connect", (error) => {
+      if (error) {
+        throw error;
+      }
+      this.setClient(new KeplrWalletConnectV1(this.connector));
+      this.ee.emit('update');
+    });
+
+    this.connector.on("disconnect", (error) => {
+      if (error) {
+        throw error;
+      }
+      this.ee.emit('disconnect');
+    });
+
+    this._client = new KeplrWalletConnectV1(this.connector);
+    this.ee = new EventEmitter();
```

Comment:
instead of `this.ee` can we can make it more declarative so future developers know what `ee` is?

I suggest `this.emitter` to be clear

Reply:
ok